app/services/action_service.rb

```diff
@@ -52,6 +52,8 @@ def remove_assigned_team(_params)
   end
 
   def send_email_transcript(emails)
+    emails = emails[0].split(',').compact
```

Comment:
Since are allowing emails as comma separated values. there could case that extra spaces could creep in. 
example : "test1@test.com.  ,        test2@test.com" . lets make sure to handle that as well. 

ref: https://github.com/chatwoot/chatwoot/commit/026e03c3073284be4e5d02b71d7027be95f9f8b2 

We could add a common filter function that we use to conversat comma separated emails string to array.